tests/base-notebook/test_packages.py

```diff
@@ -75,6 +75,7 @@
     "python",
     "r-irkernel",
     "unixodbc",
+    "pandas",
```

Comment:
You need to actually add `"pandas[version='>"` to this list.
And please, keep it sorted.